examples/prometheus/src/main.rs

```diff
@@ -51,7 +51,10 @@ fn main() {
         .source(YellowstoneGrpcSource::new())
         .source(SolanaAccountsRpcSource::new())
         .account(Pipeline::new(RaydiumAmmV4AccParser, [Logger]))
-        .instruction(Pipeline::new(RaydiumAmmV4IxParser, [Logger]))
+        .instruction(FilterPipeline::new(RaydiumAmmV4IxParser, [Logger])
+            .include_accounts(["GH8Ers4yzKR3UKDvgVu8cqJfGzU4cU62mTeg9bcJ7ug6", "4xxM4cdb6MEsCxM52xvYqkNbzvdeWWsPDZrBcTqVGUar"])
+            .required_accounts(["TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA"]),
+        )
```

Comment:
```suggestion
        .instruction(
          FilterPipeline::new(
            RaydiumAmmV4IxParser, 
            [Logger], 
            Filter::build()
              .include_accounts([
                  "GH8Ers4yzKR3UKDvgVu8cqJfGzU4cU62mTeg9bcJ7ug6",
                  "4xxM4cdb6MEsCxM52xvYqkNbzvdeWWsPDZrBcTqVGUar"
                ])
              .required_accounts(["TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA"])
          ),
        )
```

@fernandodeluret sorry I didnt mention this yesterday during our sync but looking more at the PR we are re-implementing include_accounts and required_accounts on the pipeline. In order to decouple concerns so more filter options can be added without having to adjust FIlterPipeline I think we should take a filter as the third-arguement to FilterPiplene then in its preflight it merges the filter on the parser with this extra filter. I believe Filter already supports merge.